test/unit/environment_test.rb

```diff
@@ -8,18 +8,9 @@ def setup
     end
   end
 
-  test "should have name" do
-    env = Environment.new
-    refute env.valid?
-  end
-
-  test "name should be unique" do
-    as_admin do
-      env = Environment.create :name => "foo"
-      env2 = Environment.new :name => env.name
-      refute env2.valid?
-    end
-  end
+  should validate_presence_of(:name)
+  should validate_uniqueness_of(:name)
+  should allow_value('new').for(:name)
```

Comment:
The environment model has "new" *removed* from its list of reserved words in the friendly_id call at the top of the model.  The removed test below about the name "new" is specifically to check that friendly_id doesn't prevent the environment being created.  I just wanted to check (sorry, didn't test myself) that this is still tested.